docs/content/how-to/creating-catalogs.md

```diff
@@ -175,3 +176,40 @@ Using the table option facilitates the convenient definition of Hive table param
 Parameters prefixed with `hive.` will be automatically defined in the `TBLPROPERTIES` of the Hive table. 
 For instance, using the option `hive.table.owner=Jon` will automatically add the parameter `table.owner=Jon` to the table properties during the creation process.
 
+## Creating a Catalog with JDBC Metastore
+
+By using the Paimon JDBC catalog, changes to the catalog will be directly stored in relational databases such as MySQL, postgres, etc.
```

Comment:
`postgres` is not supported for now, should we remove it?

Reply:
> `postgres` is not supported for now, should we remove it?

it's just that locking is not supported

Reply:
Can you document this? only SQLITE and MYSQL supports catalog lock